services/motion/builtin/move_request.go

```diff
@@ -133,6 +133,7 @@ func (mr *moveRequest) AnchorGeoPose() *spatialmath.GeoPose {
 // execute attempts to follow a given Plan starting from the index percribed by waypointIndex.
 // Note that waypointIndex is an atomic int that is incremented in this function after each waypoint has been successfully reached.
 func (mr *moveRequest) execute(ctx context.Context, plan motionplan.Plan) (state.ExecuteResponse, error) {
+	mr.logger.Debug("calling GoToInputs")
```

Comment:
i do not think we need this. Instead the logging could take place in the kinematic base package. This line doesn't really tell us much imo

Reply:
Ooops, this was a debug trace that got left in. Thanks for catching.